receipt_for_lovely_loveseats/checkout.py

```diff
@@ -0,0 +1,41 @@
+class Checkout:
+  def __init__(self, products = {}, items = []):
+    self.products = products
+    self.items = items
+
+  def add_to_checkout(self, product_key, count):
+    item = self.__find_item(product_key)
+
+    if item is None:
```

Comment:
From my Google-fu it seems using `is` is accepted practice in Python? In Ruby this is discouraged. Will have to find a style linter to help guide me...